File: ABC/060/B.py
import sys
A,B,C = map(int,input().split())

if A < 0 or A > 100 or B < 0 or B > 100:
    sys.exit()
if not ( C >= 0 or C < B ):
    sys.exit()

tmp = 0
result = "NO"
for I in range(B):
    tmp = A * I % B
    if tmp % B == C:
        result = "YES"

print(result)

File: ABC/ruby/060/B.rb
A,B,C = gets().chomp.split(' ').map { |n| n.to_i }
exit if !( 1 <= A && A <= 100 && 1 <= B && B <= 100 && 0 <= C && C < B)

if A % 2 == 0 && B % 2 == 0 && C % 2 == 1 
    puts 'NO'
    exit
end
for I in (1..10**5) do #繰り返しはBの回数まででOKなのでこんなにやる必要はない。。。
    if (( A * I ) % B ) == C
        puts 'YES'
        exit
    end
end
puts 'NO'
